app/__init__.py

```diff
@@ -147,7 +149,31 @@ def route(self, rule, **options):
         and POST methods, useful for form-based routes.
         """
         options.setdefault("methods", ("GET", "POST"))
-        return super().route(rule, **options)
+        
+        def decorator(func):
+            # Get function signature
+            signature = inspect.signature(func)
+            annotations = signature.parameters
```

Comment:
Avoid comments when the code is already almost English, "Get function signature" is not needed to describe `inspect.signature(func)`

Not adding comments when not needed helps with readability by reducing noise, it helps the reader to focus on what's actually important. Comments are only needed when the code is hard to understand and can not be simplified.